lib/stack_master/commands/apply.rb

```diff
@@ -64,7 +67,8 @@ def create_stack
           failed!("Stack creation aborted")
         end
         upload_files
-        cf.create_stack(stack_options.merge(tags: proposed_stack.aws_tags))
+        on_failure = @config.stack_defaults['on_failure'] || @options.on_failure
```

Comment:
Should this logic be the other way around? Take cmd line args if passed in otherwise use what's in config? If we removed L15 we could say `@options.on_failure || @config.stack_defaults['on_failure'] || 'ROLLBACK'`